Makefile

```diff
@@ -1,11 +1,23 @@
+TEST_RUN_EXTRA_PARAMS?=
+UNIT_TEST_DIR=test/test-unit
+
 default:
 	false
 
-.PHONY: lint flake8 luacheck
 lint: flake8 luacheck
 
 flake8:
 	python -m flake8 *.py lib/*.py
 
 luacheck:
 	luacheck --config .luacheckrc .
+
+build_unit_tests:
+	make -C $(UNIT_TEST_DIR)
+
+test_integration: build_unit_tests
+	test/test-run.py --builddir=build/ --var=build/test/var --force $(TEST_RUN_EXTRA_PARAMS)
+
+test: test_integration
```

Comment:
Regarding the PR description: maybe it would be better to suggest `make test` rather than direct ./test-run.py invocation with parameters?

Reply:
Updated description

BTW commit message describes `make test`:
```
commit e74f813d41c9f1cad9e4e3a84bc5b34253253e96
Author: Sergey Bronnikov <sergeyb@tarantool.org>
Date:   Fri Mar 12 14:15:44 2021 +0000

    test: add integration tests
    
    test-run supports three types of tests:
    
    - tarantool - Test-Suite for Functional Testing
    - app - Another functional Test-Suite
    - unittest - Unit-Testing Test Suite
    
    Patch adds tests for two of supported test types:
    
    - test-app for type 'app'
    - test-tarantool for type 'tarantool'
    
    How-to run:
    
    $ make test_integration
    
    - test-tarantool/panic_on_broken_lsn.test.lua [1]
    
    1. https://github.com/tarantool/tarantool-qa/issues/96
```